src/bases.d.ts

```diff
@@ -171,6 +171,11 @@ export interface StatefulMixin<S extends State>{
 	readonly state: S;
 
 	/**
+	 * A readonly reference to the stateFrom provided to the widget on instantiation.
+	 */
+	readonly stateFrom?: StoreObservablePatchable<S>;
```

Comment:
`?` implies the property may or may not be defined...  With a getter, it is always defined, but may return a value of `undefined`.  If you did `Object.keys(instance)` for example `stateFrom` would always be there.  Especially when downstream users, not using strict mode, it would more accurately represent the actual implementation.